src/blog/2024/04/displaying-logged-in-users-on-dashboard.md

```diff
@@ -0,0 +1,148 @@
+")
+
+## Installing Flowfuse user addon
+
+1. Click the Node-RED Settings (top-right)
+2. Click "Manage Palette"
+3. Switch to the "Install" tab
+4. Search for `@flowfuse/node-red-dashboard-2-user-addon`
+5. Click "Install"
+
+## Exploring the Flowfuse User Addon
+Understanding the Flowfuse User Addon can be a bit complex if you're not familiar with the basics of how it works and the type of object it attaches to the msg emitted by Dashboard 2.0 widgets. Therefore, discussing it in a separate section is important.
+The Flowfuse User Addon is a plugin developed for Dashboard 2.0, leveraging the Flowfuse API to retrieve information about logged in user. In this addon, user information is attached to the msg emitted by Dashboard 2.0 nodes. This user information object is attached as msg._client.user, Below is an example of how that object looks:
+
+```
+   "userId": "", // unique identifier for the user
+   "username": "", // FlowFuse Username
+   "email": "", // E-Mail Address connected to their FlowFuse account
+   "name": "", // Full Name
+   "image": "" // User Avatar from FlowFuse
+}
+```
+Behind the scenes, we're appending our user object to the msg object, via the SocketIO auth option. We make the socketio object available via a computed [setup](https://dashboard.flowfuse.com/contributing/guides/state-management.html#setup-store) object, this means that we can also access user data in any ui-template widget like this:
+
+```
+setup.socketio.auth.user
+```
+When running Node-RED Dashboard 2.0 on FlowFuse, you'll have a new sidebar option in the Node-RED Editor, you just have to navigate to the FF Auth tab and you’ll see two option
+
+**Option 1: Include client Data**
+
+By default, this option is enabled. When this option is enabled, an object with user information will be added to the “msg” emitted by any widget of the Node-red Dashboard 2.0.
+
+**Option 2: Accept client constraints**
+
+A feature that ensures messages are specifically targeted to individual clients, which enhances the precision and security of data transmission within the platform. It determines by enabling the nodes option in the FF Auth tab whether the enabled node type will utilize client data, such as socketid, and restrict communications to only that client.
+
+For example, consider a manufacturing facility where each production line has its own monitoring system. With this feature enabled, data from sensors on Production Line A will only be sent to the monitoring system designated for Production Line A. This ensures that data remains isolated and relevant to each specific area of operation, maintaining organizational efficiency and security.
+
+*Note: Please note that Multi-User Addons can only be used by our Teams and Enterprise Self-Hosted customers. Upon request, we provide all required configurations to get started.*
```

Comment:
@MarianRaphael Do we have an epic to make the 'upon request' go away?

Reply:
Not an epic bit an issue: https://github.com/FlowFuse/flowfuse/issues/3588